src/arch/mod.rs

```diff
@@ -16,18 +17,30 @@ pub fn get_color_convert_line_ycbcr() -> Option<unsafe fn(&[u8], &[u8], &[u8], &
             return Some(ssse3::color_convert_line_ycbcr);
         }
     }
+    // Runtime detection is not needed on aarch64.
+    #[cfg(all(feature = "aarch64_neon", target_arch = "aarch64"))]
+    {
+        return Some(neon::color_convert_line_ycbcr);
```

Comment:
Is there no  [`is_aarch64_feature_detected`](https://doc.rust-lang.org/stable/std/macro.is_aarch64_feature_detected.html) code for this?

Reply:
aarch64 makes neon mandatory, so such a function would always return true for NEON :)

Reply:
I see :joy: